R/hmde-package.R

```diff
@@ -13,6 +13,7 @@
 #' @importFrom rstantools rstan_config
 #' @importFrom RcppParallel RcppParallelLibs
 #' @importFrom stats median
+#' @importFrom deSolve ode
```

Comment:
Probably remove. If it's not essential for the package to function, then you're better off not loading it. Also update NAMESPACE

Reply:
Done.